css/main.css

```diff
@@ -400,3 +401,115 @@ button {
   top: 56px;
   gap: 8px;
 }
+
+/* styling form section */
+
+.form-section {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  background-image: url(/img/form-bg/illustration-contact-form.svg);
+  background-repeat: no-repeat;
+  background-position: 97.72% 91%;
+  width: 100%;
+  gap: 58px;
+  padding: 64px 24px 64px 24px;
+}
+
+.form-section .form-text-primary {
+  display: flex;
+  text-align: center;
+  width: 327px;
+  top: 64px;
+  left: 24px;
+  gap: 24px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+  width: 320px;
+  height: 330px;
+}
+
+form .text-field {
+  box-sizing: border-box;
+  font-size: 15px;
+  font-weight: 400;
+  left: 15px;
+  width: 327px;
+  height: 48px;
+  padding: 10px;
+  outline: none;
+  background-color: #fff;
+  border: 1px solid #d0d9d4;
+  border-radius: 4px;
+}
+
+form textarea {
+  width: 327px;
+  padding: 10px;
+  height: 114px;
+  font-size: 15px;
+  font-weight: 400;
+  border: 1px solid #d0d9d4;
+  border-radius: 4px;
+}
+
+form .submitBtn {
+  border: none;
+  color: #ffff;
+  padding: 12px;
+  gap: 10px;
+  font-weight: 500;
+  font-size: 17px;
+  text-align: center;
+  height: 48px;
+  width: 129px;
+  background: #36b37f;
+  border-radius: 4px;
+  cursor: pointer;
+  line-height: 24px;
+}
+
+input:focus,
+textarea:focus {
+  border-color: #36b37e;
+}
+
+form button:hover {
+  background: #36b37f;
+  transition: all 0.5s ease;
+  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+}
```

Comment:
Nice job adding the hover effect 😄! The requirements request for all button interactions (including the pressed). Kindly add the remaining effect to your button(s).
![image](https://user-images.githubusercontent.com/42799579/192534699-9ad31669-7b9b-4fb1-bdab-3e479b7e5123.png)

Figma design:
![image](https://user-images.githubusercontent.com/42799579/192534533-8c425811-7e34-477e-8e28-cc0c104e64c6.png)

TIP: You can add button:active to create this effect 😉.